### PR TITLE
Added caveat about the auth token

### PR DESCRIPTION
**Pull request by sascala**
@nickbp I'm not yet sure if this text can appear on the open site as well as enterprise, but either way, could you please read this over and make sure it's accurate?

### CODE REVIEW

**Comment by nickbp** [comment]
LGTM. I don't have access to merge.

**Comment by sascala** [comment]
@alberts Will an auth token be required for Open clusters at any point in the future? It's just that I'm trying to figure out if I need to create a separate uninstall page for enterprise, or if I can add this caveat about the auth token and eventually it will become relevant to an Open user.

File: 1.8/usage/managing-services/uninstall.md [hunk 1]
```diff
@@ -153,3 +153,33 @@ If you run the script via Marathon, you will also see the following output:
...
+      -p/--principal=<framework-principal>
+      -z/--zk_path=<zk-path>
+      [-m/--master_url=http://leader.mesos:5050/master/]
+      [-n/--marathon_url=http://marathon.mesos:8080/v2/apps/]
+      [-e/--exhibitor_url=http://leader.mesos:8181/]
+      [--username=dcos_user]
+      [--password=dcos_password]
+      [--auth_token=dcos_auth_tok]
+
+To get the auth token you need, run `dcos config sho core.dcos_acs_token` from the DC/OS CLI on your local machine:
```
Comment (joel-hamill) [inline 1.8/usage/managing-services/uninstall.md]: @sascala Please fix sentence grammar and change command to `dcos config show core.dcos_acs_token`

**Comment by sascala** [comment]
@joel-hamill I made those change. The "you need" wasn't necessary, but wasn't a mistake -- I meant "to get the auth token that you need". Unless you feel differently, I'd like to merge this and revisit if it turns out that the auth token caveat will never be relevant to an Open user.

**Comment by joel-hamill** [comment]
@sascala i believe `core.dcos_acs_token` is enterprise only. you should be able to verify by spinning up an open cluster and running. if so, we'll need to split this topic out between enterprise and OSS.

**Comment by sascala** [comment]
Ahh OK. I’ll test and make sure

**Comment by sascala** [comment]
Right you are:
```
calico:~ suzanne$ dcos config show core.dcos_acs_token
Property 'core.dcos_acs_token' doesn't exist
calico:~ suzanne$ 
```

**Pull request merged**